macosx/PrefsController.m

```diff
@@ -845,7 +845,8 @@ - (void) folderSheetShow: (id) sender
             [fDefaults setBool: YES forKey: @"DownloadLocationConstant"];
             [self updateShowAddMagnetWindowField];
             
-            tr_sessionSetDownloadDir(fHandle, [folder UTF8String]);
+            assert(folder.length > 0);
```

Comment:
I agree in general. I did not want to make things more complicated than necessary for the moment. As the path comes from the panel, it’s unlikely to be empty. So it’s just defensive programming. Please suggest a better solution. :)